joplin/api/schema.py

```diff
@@ -227,6 +289,10 @@ class Meta:
         fields = '__all__'
         interfaces = [graphene.Node, DepartmentResolver]
 
+    @superuser_required
```

Comment:
I got tempted to try to clean up the copypasta here with multiple inheritance but that's a v3 decision

Reply:
A good impulse. And yeah it would be much easier to manage in v3.